### ISSUE TITLE
How to enable Optimus again... blindly.

### ISSUE THREAD

**Issue by ShiftZ** [seerge/g-helper#1668]
I flashed a wrong BIOS to my dGPU on Asus X16 Flow. It worked ok until I switched to "Ultimate GPU mode" in GHelper, and now it's black screen. Windows works but I can't see anything. I can't switch to iGPU back. I disassembled my laptop, reseted BIOS, but it just doesn't work. I tried to flash the factory BIOS back blindly, but I failed. I need my iGPU working to try different things.
So, there are two questions:
a) Where is that MUX switch flag physically? Is it a bios setting? By resetting the BIOS shouldn't I get the Optimus working again?
b) How can I switch to Eco GPU mode (no dGPU) using G-Helper, but without screen. Is there a command line for that. 
c) Given that I'm a programmer but not C#, unfortunately, do you think it would be hard to implement the command line?

I have another laptop, I can write batch files to a USB flash drive and execute them blindly on the broken laptop. I can then write the output of the commands in a file and read it.

**Comment by seerge** [comment]
@ShiftZ hello, if you want to restore mux switch into it's standard state (i.e. Standard mode), you can just do a hardware reset
https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop

Alternatively you can just connect external monitor and operate app

**Comment by ShiftZ** [comment]
Hardware reset unfortunately, doesn't work.

**Comment by seerge** [comment]
Then it's probably some other issues, due to flashed bios.

You can also turn on mux switch via powershell as admin

```
(Get-WmiObject -Namespace Root/WMI -Class AsusAtkWmi_WMNB).DEVS(0x00090016, 1)
```

**Comment by ShiftZ** [comment]
Thanks. That probably would've helped me, but I was able to resolve the
issue by blindly starting windows in Safe Mode and flashing it back to the
original BIOS.
The funny thing is that my laptop is still in Ultimate GPU mode. I did
hardware reset several times. I disassembled my laptop completely. I
removed the battery. I was looking for a CMOS battery, but it turned out
there's no CMOS battery at all. I had the motherboard completely removed
from the chassis, I tried hardware reset on a demontaged naked motherboard,
and somehow the damn switch is still there!
